valhalla/filesystem.h

```diff
@@ -93,6 +110,20 @@ class path {
   std::vector<size_t> separators_;
 };
 
+template <class CharT, class Traits>
+std::basic_ostream<CharT, Traits>& operator<<(std::basic_ostream<CharT, Traits>& os, const path& p) {
+  os << p.string();
+  return os;
+}
+
+template <class CharT, class Traits>
+std::basic_istream<CharT, Traits>& operator>>(std::basic_istream<CharT, Traits>& is, path& p) {
+  std::string tmp;
+  is >> tmp;
+  p = path(tmp);
+  return is;
+}
```

Comment:
need sometihng like: https://github.com/valhalla/valhalla/pull/2008/files on these to handle spaces in the input